components/director/pkg/str/strings.go

```diff
@@ -38,3 +40,12 @@ func Cast(i interface{}) (string, error) {
 
 	return "", apperrors.NewInvalidStringCastError()
 }
+
+func PrefixStrings(in []string, prefix string) []string {
```

Comment:
What do you think about using `variadic` arg?
It will be easier nicer to use.

Reply:
I'm not convinced it'll be nicer. We use slice of strings as an input all the times anyway, and also we have to return a slice, so the input and output format is unified.